.github/workflows/docker_build.yml

```diff
@@ -150,18 +125,8 @@ jobs:
             OMNI_COMPONENT_COMMIT=${{ github.sha }}
             OMNI_COMPONENT_VERSION=${{ steps.meta.outputs.version }}
 
-      - name: save docker image to tar file
-        if: env.MINIO_ENABLE
-        run: docker save --output ./${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }} ${{ steps.meta.outputs.tags }}
-
-      - name: upload docker image to private minio storage
-        if: env.MINIO_ENABLE
-        run: |
-          export AWS_ACCESS_KEY_ID=${{ secrets.MINIO_ACCESS_KEY_ID }}
-          export AWS_SECRET_ACCESS_KEY=${{ secrets.MINIO_SECRET_ACCESS_KEY }}
-          aws --endpoint-url ${{ secrets.MINIO_ENDPOINT_URL }} s3 cp  ./${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}  s3://docker-img-artifacts/${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}
-
     outputs:
-      image_tar_name: ${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}
+      docker_image_fullname: ${{ steps.meta.outputs.tags }}
```

Comment:
now you are using the tags. then let's update the `docker_image_fullname` to `docker_tags`